dbt/adapters/rockset/impl.py

```diff
@@ -73,7 +75,13 @@ def date_function(cls):
     def create_schema(self, relation: RocksetRelation) -> None:
         rs = self._rs_client()
         logger.debug('Creating workspace "{}"', relation.schema)
+        # Must check if the workspace already exists...
+        current_workspaces = {ws.name for ws in rs.Workspaces.list().data}
+        if relation.schema in current_workspaces:
+            return
         rs.Workspaces.create(name=relation.schema)
+        # Wait for workspace creation to complete
+        sleep(2)
```

Comment:
I see we were already using just sleep to make sure resources are created but is there a reason to not make a get call?

Reply:
Fixed